modules/sdk-rpc-wrapper/src/evmRPCWrapper.ts

```diff
@@ -51,4 +67,28 @@ export class EvmRPCWrapper {
       result,
     };
   }
+
+  private async sendTransaction(options: EVMRPCTransactionOptions): Promise<any> {
+    const { to, data, gasPrice, gasLimit, value } = options;
+
+    const sendManyOptions: SendManyOptions = {
+      recipients: [
```

Comment:
I'm assuming `sendTransaction` is for sending 1 transaction - why is `recipients` a list here?

Reply:
[SenMany Method](https://github.com/BitGo/BitGoJS/blob/master/modules/sdk-core/src/bitgo/wallet/wallet.ts#L1989)

[SendManyOptions](https://github.com/BitGo/BitGoJS/blob/master/modules/sdk-core/src/bitgo/wallet/iWallet.ts#L447)